--- harmonic_gravity_complex_3.cpp	2020-06-01 19:41:35.046160674 -0700
+++ harmonic_gravity_complex_3_patched.cpp	2020-06-01 19:41:01.681639205 -0700
@@ -227,8 +227,11 @@
 
     /* Warn if inside the reference radius */
     if ( std::real(r) < refRad )    {
-        mexWarnMsgTxt( "HARMONIC_GRAVITY.DLL - Computing gravity inside "
-                       "the spherical harmonic reference radius" );
+        // mexWarnMsgTxt( "HARMONIC_GRAVITY.DLL - Computing gravity inside "
+        //                "the spherical harmonic reference radius" );
+        mexWarnMsgIdAndTxt("SBDT:harmonic_gravity:inside_radius",
+            "HARMONIC_GRAVITY.DLL - Computing gravity inside "
+            "the spherical harmonic reference radius" );
     }
  
    
